tools/linter_lib/custom_check.py

```diff
@@ -489,16 +505,24 @@ def build_custom_checkers(by_lang):
                          "static/templates/settings/bot-settings.handlebars",
                          "templates/zerver/email_log.html"])},
         {'pattern': "placeholder='[^{]",
-         'description': "`placeholder` value should be translatable."},
+         'description': "`placeholder` value should be translatable.",
+         'good_lines': ["<input placeholder='{foo()}'></input>"],
+         'bad_lines': ["<input placeholder='foo'></input>"]},
         {'pattern': "aria-label='[^{]",
-         'description': "`aria-label` value should be translatable."},
+         'description': "`aria-label` value should be translatable.",
+         'good_lines': ["<button aria-label='{foo(){}'></button>"],
+         'bad_lines': ["<button aria-label='foo'></button>"]},
         {'pattern': 'aria-label="[^{]',
          'description': "`aria-label` value should be translatable.",
          'exclude': set(["docs/_templates/layout.html"])},
         {'pattern': 'script src="http',
-         'description': "Don't directly load dependencies from CDNs.  See docs/subsystems/front-end-build-process.md"},
+         'description': "Don't directly load dependencies from CDNs.  See docs/subsystems/front-end-build-process.md",
+         'good_lines': ['<script src="jquery.min.js"></script>'],
```

Comment:
That line won't actually work.  We actually do all of our JS via bundles managed by webpack. `{{ render_bundle('translations') }}` is a reasonable `good_lines` example.